modules/programs/neomutt.nix

```diff
@@ -266,6 +267,11 @@ in {
         description = "Extra configuration appended to the end.";
       };
 
+      changeFolderWhenSourcingAccount =
```

Comment:
I don't really like this name since it's pretty long, but I couldn't think of anything better. Suggestions welcome!